diff --git a/node_modules/@ittai/builder/build/index.js b/node_modules/@ittai/builder/build/index.js
index 850ce76..41b0505 100644
--- a/node_modules/@ittai/builder/build/index.js
+++ b/node_modules/@ittai/builder/build/index.js
@@ -278,11 +278,12 @@ function build(argv, forceNoWatch = false) {
 								} else {
 									const localeOpts = { minimumIntegerDigits: 2, useGrouping: false }
 									const currentTime = new Date()
+									const lang = Intl.DateTimeFormat().resolvedOptions().locale;
 									logger.makeBadge("✓", term.bold.bgGreen)
 										("Build done! Built in ")
 										.brightGreen(`${logger.nf`\ufa19`}${Math.round((nanoseconds() - startTime) / 1000000).toLocaleString()} ms`)
 										(`${(!logger.nf() ? " " : "")} at `)
-										.cyan(`${logger.nf`\ufa1a`}${currentTime.getHours().toLocaleString(undefined, localeOpts)}:${currentTime.getMinutes().toLocaleString(undefined, localeOpts)}:${currentTime.getSeconds().toLocaleString(undefined, localeOpts)}`)
+										.cyan(`${currentTime.getHours().toLocaleString(lang, localeOpts)}:${currentTime.getMinutes().toLocaleString(lang, localeOpts)}:${currentTime.getSeconds().toLocaleString(lang, localeOpts)}`)
 										(".\n\n")
 								}
 							}
