diff --git a/node_modules/@ittai/builder/build/index.js b/node_modules/@ittai/builder/build/index.js
index 9823519..16181d4 100644
--- a/node_modules/@ittai/builder/build/index.js
+++ b/node_modules/@ittai/builder/build/index.js
@@ -262,11 +262,12 @@ function build(argv, forceNoWatch = false) {
 								} else {
 									const localeOpts = { minimumIntegerDigits: 2, useGrouping: false }
 									const currentTime = new Date()
+									const lang = Intl.DateTimeFormat().resolvedOptions().locale;
 									logger.makeBadge("✓", term.bold.bgGreen)
 										("Build done! Built in ")
 										.brightGreen(`${Math.round((nanoseconds() - startTime) / 1000).toLocaleString()} ms`)
 										(" at ")
-										.cyan(`${currentTime.getHours().toLocaleString(process.env.LANG, localeOpts)}:${currentTime.getMinutes().toLocaleString(process.env.LANG, localeOpts)}:${currentTime.getSeconds().toLocaleString(process.env.LANG, localeOpts)}`)
+										.cyan(`${currentTime.getHours().toLocaleString(lang, localeOpts)}:${currentTime.getMinutes().toLocaleString(lang, localeOpts)}:${currentTime.getSeconds().toLocaleString(lang, localeOpts)}`)
 										(".\n\n")
 								}
 							}
@@ -370,7 +371,10 @@ function build(argv, forceNoWatch = false) {
 										loader: require.resolve("css-loader"),
 										options: {
 											modules: true,
-											import: true
+											import: true,
+											modules: {
+												localIdentName: "[local]",
+											}
 										},
 									},
 								],
@@ -384,7 +388,10 @@ function build(argv, forceNoWatch = false) {
 										options: {
 											modules: true,
 											import: true,
-											importLoaders: 1
+											importLoaders: 1,
+											modules: {
+												localIdentName: "[local]",
+											}
 										},
 									},
 									require.resolve("sass-loader"),
@@ -399,7 +406,10 @@ function build(argv, forceNoWatch = false) {
 										options: {
 											modules: true,
 											import: true,
-											importLoaders: 1
+											importLoaders: 1,
+											modules: {
+												localIdentName: "[local]",
+											}
 										},
 									},
 									require.resolve("stylus-loader"),
@@ -414,7 +424,10 @@ function build(argv, forceNoWatch = false) {
 										options: {
 											modules: true,
 											import: true,
-											importLoaders: 1
+											importLoaders: 1,
+											modules: {
+												localIdentName: "[local]",
+											}
 										},
 									},
 									require.resolve("less-loader"),
