diff --git a/node_modules/ittai/entities/Plugin.js b/node_modules/ittai/entities/Plugin.js
index b109a56..b768ec2 100644
--- a/node_modules/ittai/entities/Plugin.js
+++ b/node_modules/ittai/entities/Plugin.js
@@ -78,6 +78,10 @@ export default /**
 			hasSettingsPanel = false;
 			cachedCss = [];
 
+			load() {
+				if (this.load) this.instance.load();
+			}
+
 			start() {
 				try {
 					this.cachedCss.forEach((e, k) => {
diff --git a/node_modules/ittai/logger/createArguments.js b/node_modules/ittai/logger/createArguments.js
index 3a4499c..2358a96 100644
--- a/node_modules/ittai/logger/createArguments.js
+++ b/node_modules/ittai/logger/createArguments.js
@@ -1,6 +1,6 @@
 export default function createArguments(...args) {
 	return [
-		"%cIttai",
+		"%cFosscord",
 		"color: #000; background-color: #42ffa7; font-family: default; padding-left: 3px; padding-right: 3px; border-radius: 2px; font-weight: bold;",
 		...args,
 	];
diff --git a/node_modules/ittai/patcher/index.d.ts b/node_modules/ittai/patcher/index.d.ts
index ff0d174..b9e677d 100644
--- a/node_modules/ittai/patcher/index.d.ts
+++ b/node_modules/ittai/patcher/index.d.ts
@@ -1,17 +1,17 @@
-type GenericPatchFunction = (args: any, thisArg: any) => any
-type AfterPatchFunction = (args: any, res: any, thisArg: any) => any
+type GenericPatchFunction = (args: any, original: any, thisArg: any) => any;
+type AfterPatchFunction = (args: any, res: any, thisArg: any) => any;
 
 type PatchData = {
-    name: string
-    type: string
-    patchFunction: GenericPatchFunction | AfterPatchFunction
-    unpatch: () => void
-}
+	name: string;
+	type: string;
+	patchFunction: GenericPatchFunction | AfterPatchFunction;
+	unpatch: () => void;
+};
 
-export function patch(name: string, object: any, functionName: string, type: "after" | "before" | "instead", patchFunction: GenericPatchFunction | AfterPatchFunction): PatchData
+export function patch(name: string, object: any, functionName: string, type: "after" | "before" | "instead", patchFunction: GenericPatchFunction | AfterPatchFunction): PatchData;
 
-export function after(name: string, object: any, functionName: string, patchFunction: AfterPatchFunction): PatchData
-export function before(name: string, object: any, functionName: string, patchFunction: GenericPatchFunction): PatchData
-export function instead(name: string, object: any, functionName: string, patchFunction: GenericPatchFunction): PatchData
+export function after(name: string, object: any, functionName: string, patchFunction: AfterPatchFunction): PatchData;
+export function before(name: string, object: any, functionName: string, patchFunction: GenericPatchFunction): PatchData;
+export function instead(name: string, object: any, functionName: string, patchFunction: GenericPatchFunction): PatchData;
 
-export function unpatchAll(unpatches?: string[]): void
\ No newline at end of file
+export function unpatchAll(unpatches?: string[]): void;
\ No newline at end of file
diff --git a/node_modules/ittai/patcher/patch.js b/node_modules/ittai/patcher/patch.js
index 8b1b09c..a1c9b04 100644
--- a/node_modules/ittai/patcher/patch.js
+++ b/node_modules/ittai/patcher/patch.js
@@ -12,6 +12,8 @@ export default /**
  * @memberof module:patcher
  * @tutorial patching
  */ function patch(name, object, functionName, type, patchFunction) {
+	if (!object) throw new Error("Cannot patch nothing!");
+
 	const id = object.__ittai__ ?? randomString(25, Object.keys(patches));
 	object.__ittai__ = object.__ittai__ ?? id;
 	if (!patches[id]) patches[id] = {};
@@ -55,6 +57,10 @@ export default /**
 			patches: [],
 		};
 
+		if (!patches[id][functionName].original) {
+			throw new Error(`original function ${functionName} lost!`);
+		}
+
 		const props = { ...object[functionName] };
 
 		object[functionName] = function (...args) {
@@ -67,7 +73,7 @@ export default /**
 			for (const before of befores) {
 				try {
 					const callback = before.patchFunction(args, this);
-					if (callback) args = callback
+					if (callback) args = callback;
 				} catch (e) {
 					logger.error(`Error running before patch ${name}.`, e);
 				}
@@ -83,10 +89,7 @@ export default /**
 				for (const instead of insteads) {
 					// Do trash merge with Lodash.
 					try {
-						(res = globalThis._.merge(
-							res,
-							instead.patchFunction(args, this) ?? {}
-						)),
+						(res = instead.patchFunction(args, functionData.original.bind(this), this)),
 							(ranOnce = true);
 					} catch (e) {
 						logger.error(`Error running instead patch ${name}.`, e);
@@ -100,8 +103,8 @@ export default /**
 			// After patches.
 			for (const after of afters) {
 				try {
-					const callback = after.patchFunction(args, res, this)
-					if (callback) res = callback
+					const callback = after.patchFunction(args, res, this);
+					if (callback) res = callback;
 				} catch (e) {
 					logger.error(`Error running after patch ${name}.`, e);
 				}
